### PR TITLE
project setup

### PR DESCRIPTION
**Pull request by ShrutiSachpara**

### CODE REVIEW

File: app/helper/db.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const db = {};
+db.Sequelize = Sequelize;
+db.sequelize = sequelize;
+
+db.authUserModel = require('../model/user')(sequelize, Sequelize);
+
+db.sequelize.sync().then(() => {
+  logger.info('yes re-sync');
+}).catch((err) => {
+  console.log('no sync error', err);
```
Comment (prajapatiRgithub) [inline app/helper/db.js]: Use logger here.

File: app/helper/multer.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const multer = require("multer");
```
Comment (prajapatiRgithub) [inline app/helper/multer.js]: Use single quotation in entire file.

File: app/model/user.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+module.exports = (sequelize, Sequelize) => {
```
Comment (prajapatiRgithub) [inline app/model/user.js:1]: what's the meaning of this model.

Comment (ShrutiSachpara) [reply app/model/user.js:1]: i have put for the testing purpose only for the check

**Review by prajapatiRgithub — APPROVED** [review]

**Pull request merged**